ts/Core/Axis/Axis.ts

```diff
@@ -6452,7 +6452,7 @@ class Axis implements AxisComposition, AxisLike {
             step,
             bestScore = Number.MAX_VALUE,
             autoRotation: any,
-            range = (this.max as any) - (this.min as any),
+            range = Math.abs((this.max as any) - (this.min as any)),
```

Comment:
I'm not sure I understand why a negative value should be turned positive. Isn't the point to keep it above 0 so that we don't enter an eternal loop? In that case, this would be more logically correct (and easier to understand for future us):

`range = Math.max((this.max as any) - (this.min as any), 0),`

Reply:
Yep, you're right, that makes much more sense!